src/ChurchCRM/utils/LoggerUtils.php

```diff
@@ -36,6 +36,16 @@ public static function buildLogFilePath($type)
         return $logFilePrefix = SystemURLs::getDocumentRoot() . '/logs/' . date("Y-m-d") . '-' . $type . '.log';
     }
 
+    /**
+     * @return Logger
+     */
+    public static function getSlimMVCLogger() {
```

Comment:
We can remove the docblock and add this to the function signature:

```suggestion
    public static function getSlimMVCLogger(): Logger {
```